README.md

```diff
@@ -4,5 +4,12 @@ This is a repository for useful methods and circe decoders. These can be used wh
 
 * S3 Event Decoder - The messages we get through  in a Lambda are an S3 message inside an SNS message inside an SQS message. This method returns all S3 events from the message along with a list of errors, if any.
 * AWS Decoders - Circe decoders for working with the above classes which are used by the event decoder method. These are needed because circe won't auto decode case classes.
+* SQS Utils - There is a send and a delete method for sqs messages.
+* SES Utils - There is a send email method.
+
+There are also clients for AWS services. These are configured with an http client, the region and and an endpoint configurable in an aplication.conf file. There are:
```

Comment:
Double "and": "... the region and and an endpoint ..."

Reply:
I think I was getting warnings about that before I switched them off thinking I know better :smile: